doc/faq.md

```diff
@@ -1,122 +1,80 @@
 
-By default, LXD has no password for security reasons, so you can't do a remote
-add this way. To set a password, enter the following command on the host LXD is
-running on:
+Almost everything can be run in an unprivileged container, or - in cases of things that require unusual privileges, like wanting to mount NFS file systems inside the container - you might need to use bind mounts.
 
-```bash
-lxc config set core.trust_password SECRET
-```
+## Can I bind-mount my home directory in a container?
 
-This will set the remote password that you can then use to do `lxc remote add`.
+Yes, you can do this by using a {ref}`disk device <devices-disk>`:
 
-You can also access the server without setting a password by copying the client
-certificate (`~/.config/lxc/client.crt` or `~/snap/lxd/common/config/client.crt`
-for snap users) to the server and adding it with:
+    lxc config device add container-name home disk source=/home/${USER} path=/home/ubuntu
 
-```bash
-lxc config trust add client.crt
-```
+For unprivileged containers, you need to make sure that the user in the container has working read/write permissions.
+Otherwise, all files will show up as the overflow UID/GID (`65536:65536`) and access to anything that's not world-readable will fail.
+Use either of the following methods to grant the required permissions:
 
-See {doc}`authentication` for detailed information.
-
-### Can I bind-mount my home directory in a container?
-
-Yes. This can be done using a disk device:
-
-```bash
-lxc config device add container-name home disk source=/home/${USER} path=/home/ubuntu
-```
-
-For unprivileged containers, you will also need one of:
-
-- Pass `shift=true` to the `lxc config device add` call. This depends on shiftfs being supported (see `lxc info`)
-- `raw.idmap` entry (see [Idmaps for user namespace](userns-idmap.md))
-- Recursive POSIX ACLs placed on your home directory
-
-Either of those can be used to allow the user in the container to have working read/write permissions.
-When not setting one of those, everything will show up as the overflow UID/GID (65536:65536)
-and access to anything that's not world readable will fail.
+- Pass `shift=true` to the `lxc config device add` call. This depends on shiftfs being supported (see `lxc info`).
+- Add a `raw.idmap` entry (see [Idmaps for user namespace](userns-idmap.md)).
+- Place recursive POSIX ACLs on your home directory.
 
 Privileged containers do not have this issue because all UID/GID in the container are the same as outside.
 But that's also the cause of most of the security issues with such privileged containers.
 
-### How can I run Docker inside a LXD container?
+## How can I run Docker inside a LXD container?
 
-To run Docker inside a LXD container, the `security.nesting` property of the container should be set to `true`.
+To run Docker inside a LXD container, set the [`security.nesting`](instance-options-security) property of the container to `true`:
```

Comment:
Can we add a link to https://www.youtube.com/watch?v=_fCSSEyiGro if not already?

Reply:
Nice! Added.